source/clear-linux/guides/telemetrics/telem-guide.rst

```diff
@@ -378,7 +491,8 @@ Confirm that the telemetrics header file is located on the system at :file:`usr/
     #include <telemetry.h>
 
 
-   Use the following code to create the variables we need to hold the data for the record we will be creating:
+   Use the following code to create the variables needed to hold the data for
+   the record to be created:
 
    ::
```

Comment:
Wherever, you see `::` in this document, please replace it with our standard 

"""
.. code-block:: console
   
   my_code_here 
   ....      
 """

This document follows the above convention until Ln 449; thereafter, it changes to use `::`.

Reply:
I'll fix it and commit ASAP.

Reply:
Replaced old code notation using double colon (::) with code-block console notation.